docs/github.md

```diff
@@ -112,6 +129,18 @@ $ # Push the updated branch to the remote. Jujutsu automatically makes it a forc
 $ jj git push --branch your-feature
 ```
 
+## Working with other people's branches
+
+By default `jj git clone` and `jj git fetch` clone all active branches from
+the remote, this means that if you want to iterate or test another 
+contributors branch you can either `jj new <branchname>` or 
+`jj edit <branchname>`.
+
+If your remote has a large amount of old, inactive branches or this feature is
+undesirable, set `git.auto-local-branch = false` in the config file.
+
+More you can find more information on that setting [here][auto-branch].
+
```

Comment:
@joyously Is this good enough to address your questions about branches?

Reply:
That's better than nothing, thanks.